doc/how_to_guides/parallel_planning/parallel_planning_tutorial.rst

```diff
@@ -0,0 +1,174 @@
+Running Multiple Planning Pipelines in Parallel with MoveItCpp
+==============================================================
+
+MoveItCpp offers an API that allows:
+
+1. Running multiple planning pipelines in parallel with different planners
+2. Defining a custom stopping criterion that terminates the pipelines that haven't found a solution
+3. Defining a custom function to select the most suitable solutions
+
+Using multiple pipelines can be beneficial for several reasons, including:
+
+- The planner that will produce the best solution is not known a priori
+- There is a chance that the preferred planner might fail and a backup solution should be available
+
+A general introduction to MoveItCpp can be found in the :doc:`/doc/examples/moveit_cpp/moveitcpp_tutorial`.
+
+Parallel Planning Interface
+---------------------------
+
+Using parallel planning with MoveItCpp is very similar to single pipeline planning, except that a different implementation
+of the planning component's :code:`plan(...)` function is used:
+
+.. code-block:: c++
+
+    MotionPlanResponse PlanningComponent::plan(
+      const MultiPipelinePlanRequestParameters& parameters,
+      SolutionCallbackFunction solution_selection_callback,
+      StoppingCriterionFunction stopping_criterion_callback)
+
+This function tries to plan a trajectory from a start state to a goal state that satisfies a set of constraints. Based on the configuration
+provided by the :code:`parameters`, multiple threads are launched and each tries to solve the planning problem with a different planning pipeline. Once
+all pipelines have been terminated. Please keep in mind, that o solution is also a possible result, the :code:`solution_selection_callback` is called to determine which
+solution is returned as :code:`MotionPlanResponse`. By default, all pipelines use their time budget defined by the :code:`planning_time` field of the :code:`PlanRequestParameters`, but it is possible to terminate the parallel planning earlier by using the :code:`stopping_criterion_callback`. This function
```

Comment:
```suggestion
solution is returned as :code:`MotionPlanResponse`. By default, all pipelines use their time budget defined by the :code:`planning_time` field of the :code:`MultiPipelinePlanRequestParameters`, but it is possible to terminate the parallel planning earlier by using the :code:`stopping_criterion_callback`. This function
```

or you could refer to it simply as `parameters` as you do earlier in the paragraph. As long as it's consistent.